src/modules/micrortps_bridge/micrortps_client/microRTPS_client_main.cpp

```diff
@@ -100,6 +107,16 @@ static int parse_options(int argc, char *argv[])
 		}
 	}
 
+	if (_options.sleep_ms < 1) {
+		_options.sleep_ms = 1;
+		PX4_ERR("sleep time too low, using 1 ms");
+	}
+
+	if (_options.poll_ms < 1) {
+		_options.poll_ms = 1;
+		PX4_ERR("poll timeout too low, using 1 ms");
+	}
+
 	return 0;
```

Comment:
Yes, this should definitely be reworked. There should *never* be any usleep in the code execution path.